luxtronik/discover.py

```diff
@@ -0,0 +1,56 @@
+"""Discovery funtion for Luxtronik2 heatpump controllers."""
+import logging
+import socket
+
+LOGGER = logging.getLogger("Luxtronik.Discover")
+
+
+def discover() -> list[(str, int)]:
+    """Broadcast discovery for luxtronik heatpumps."""
+
+    results: list[(str, int)] = list()
+
+    for magic_port in (4444, 47808):
```

Comment:
I would suggest to put those ports into a constant, e.g. something like `LUXTRONIK_DISCOVERY_PORTS`. That way the code is better readable, the values could be re-used and changed later on easily.